src/api/server/common.rst

```diff
@@ -1210,9 +1215,11 @@ error.
     :>json string target: Replication target
     :>json string start_time: Timestamp of when the replication was started
     :>json string last_update: Timestamp of last state update
-    :>json object info: May contain additional information about the state.
-                        For error states, this will be a string. For success
-                        states this will contain a JSON object (see below).
+    :>json object info: May contain additional information about the
+                        state. For error states, this will be an
+                        object with an `"error"` field and a string
+                        value. For success states this will contain a
+                        JSON object (see below).
```

Comment:
After editing the 'error' case, the success case seems redundant. Perhaps reword to something like this?

> Will contain additional information about the state. For errors, this will be an object with an `"error`" field and string value. For success states, see below.

Reply:
That's much better. Thank you!